images/microOS/config.xml

```diff
@@ -1,116 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- OBS-Profiles: @BUILD_FLAVOR@ -->
-<image schemaversion="6.7" name="project-aquarium-microOS" displayname="Project Aquarium's MicroOS">
+<image schemaversion="6.8" name="project-aquarium-microOS" displayname="Project Aquarium's MicroOS">
     <description type="system">
-        <author>openSUSE Project</author>
-        <contact>opensuse-factory@opensuse.org</contact>
-        <specification>openSUSE MicroOS</specification>
+        <author>Aquarium Project</author>
+        <contact>alau@suse.com</contact>
+        <specification>
+            Aquarium Appliance, is a up and run storage image that replace ceph distribution.
```

Comment:
Instead of `replace ceph distribution` I'd say it `wraps around Ceph`.

Reply:
Sure, just to replacing microOS to something closer to what we are actually building.